tools/scripts/Makefile

```diff
@@ -1,7 +1,14 @@
-export PROCESSOR_CORE 
+#------------------------------------------------------------------------------
```

Comment:
this modification is not related to altera build, i would do it in a separate commit